netlify-functions-ecommerce/seed.js

```diff
@@ -5,8 +5,8 @@ const config = require('./.config');
 const mongoose = require('./mongoose');
 
 async function createProducts() {
-  await mongoose.connect(config.mongodbUri);
-  await mongoose.connection.dropDatabase();
+  await mongoose.connect(config.astraUri);
+  //await mongoose.connection.dropDatabase();//dropDatabase is not implemented by stargate-mongoose at the moment.
```

Comment:
Can we replace this with `await Product.deleteMany()`, or `await Promise.all(Object.values(mongoose. models).map(Model => Model.deleteMany())`? I can confirm that [dropDatabase is not implemented](https://github.com/stargate/stargate-mongoose/blob/c63ab1aa6fc1bb825210f2fe967fad216a55e167/src/collections/db.ts#L106-L111), so we do need this change, but the purpose of `dropDatabase()` here is to clear out any conflicting data and avoid duplicate products.

Reply:
Makes sense, let me make that change, test again and merge this.

Reply:
added createCollection (in case if it doesn't exist) and deleteMany to clean up all records before loading data